configs/standard-1year-colorado.yml

```diff
@@ -1,6 +1,6 @@
 # Only include counties within this state (give as 2 digit FIPS)
 # "10" for Delaware (3 counties), "08" for Colorado (64 counties)
-filterstate: null
```

Comment:
Looks like the original standard-1year.yml file got removed.  Please put it back.

Reply:
done